requirements.txt

```diff
@@ -0,0 +1,5 @@
+lsst-felis
```

Comment:
Use the git+https form for this file so that actions can pick up current main and not pypi version. Also for lsst-utils. This will match jenkins behavior.